src/deepsparse/image_classification/schemas.py

```diff
@@ -38,10 +38,15 @@ class ImageClassificationOutput(BaseModel):
     """
     Output model for image classification
     """
+    ndarray: numpy.ndarray = numpy.ndarray
```

Comment:
I'd say this adds unnecessary clutter. Let's just use for typing `numpy.ndarray`